automatic/sqlite/tools/chocolateyInstall.ps1

```diff
@@ -1,23 +1,24 @@
 ﻿$ErrorActionPreference = 'Stop'
 
 $toolsDir = Split-Path $MyInvocation.MyCommand.Definition
-$embedded_path = if ((Get-OSArchitectureWidth 64) -and $env:chocolateyForceX86 -ne 'true') {
-         Write-Host "Installing 64 bit version"; Get-Item "$toolsDir\*dll*win64*.zip"
-} else { Write-Host "Installing 32 bit version"; Get-Item "$toolsDir\*dll*win32*.zip" }
 
 $packageArgs = @{
-    PackageName  = 'sqlite'
-    FileFullPath = $embedded_path
-    Destination  = $toolsDir
+    PackageName    = 'sqlite'
+    FileFullPath   = (Get-Item "$toolsDir\*dll*win-x86*.zip")
+    FileFullPath64 = (Get-Item "$toolsDir\*dll*win-x64*.zip")
```

Comment:
We're not accepting `Get-Item` for packages. Can you please update that to have the explicit filename (which can be embedded in the script from the update script)?

Reply:
Removed the use of `Get-Item` - replaced with explicit embedded filenames.